<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>helpAboutUiMagician</class>
 <widget class="QDialog" name="helpAboutUiMagician">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About UI Magician</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../AutosResourceFile.qrc">
    <normaloff>:/Images/Images/fatbee_v2.ico</normaloff>:/Images/Images/fatbee_v2.ico</iconset>
  </property>
  <widget class="QLabel" name="helpImageLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>451</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="AboutUIhorizontalLayout">
    <item>
     <spacer name="AboutUIhorizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>268</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="closePushButton">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="helpImageTextVscpVersion">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Version</string>
   </property>
  </widget>
  <widget class="QLabel" name="helpImageTextAuthor">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
  </widget>
  <widget class="QLabel" name="helpImageTextCopyright">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy Right</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>helpImageLabel</zorder>
  <zorder>helpImageTextVscpVersion</zorder>
  <zorder>helpImageTextAuthor</zorder>
  <zorder>helpImageTextCopyright</zorder>
 </widget>
 <resources>
  <include location="../AutosResourceFile.qrc"/>
 </resources>
 <connections/>
</ui>
